python/code_challenges/hashtable_repeated_word.py

```diff
@@ -1,5 +1,13 @@
-from data_structures.hashtable import Hashtable
+import re
 
+def first_repeated_word(word):
+    regex = re.compile('[^a-zA-Z ]')
+    stripped_words = regex.sub('', word)
+    words = stripped_words.lower().split()
 
-def first_repeated_word():
-    pass
+    dictionary = set()
```

Comment:
Similar idea to above choosing variable names that maximize readability for `dictionary`. Maybe `words_checked`?